R/automateDataPreparation.R

```diff
@@ -7,6 +7,22 @@ automateDataPreparation <- function(datList = NULL, inputList, path = NULL,
 						addLeadingZeros=FALSE, truncateSpaceChar = TRUE, newID = NULL, oldIDs = NULL,
             missing.rule = list(mvi = 0, mnr = 0, mci = NA, mbd = NA, mir = 0, mbi = 0), verbose=FALSE) {
 
+  checkmate::assert_list(datList, types = "data.frame", null.ok = TRUE)
+  checkmate::assert_list(inputList, types = "data.frame")
+  checkmate::assert_character(path, len = 1, null.ok = TRUE)
```

Comment:
`path` has `NULL` as default, but in man page line 24 it says 'current working directory' is the default. I you want, I could correct that, but you'd have to tell me which one is correct.